lesson14_homework_interface/homework_14.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+
+class BaseHandler:
+    def __init__(self, path):
+        self.__path = path
+        self.content = None
+        self._file = None
+
+    @property
+    def path(self):
+        return self.__path
+
+    def read(self):
+        raise NotImplementedError
+
+    def append(self, new_string):
+        raise NotImplementedError
+
+    def close(self):
+        raise NotImplementedError
+
+
+class JsonHandler(BaseHandler):
+    def __init__(self, path):
+        super().__init__(path)
+        try:
+            with open(self.path, 'r') as self._file:
+                self.content = json.load(self._file)
+            quit('this json file format is not supported') if not isinstance(self.content, list) else ...
+        except FileNotFoundError:
```

Comment:
I think better to raise this exception - because this is File HANDLER.. And this name looks like this class will only handle existing files